static-build/qemu-virtiofs/Dockerfile

```diff
@@ -0,0 +1,50 @@
+from ubuntu:18.04
+
+ARG QEMU_VIRTIOFS_REPO
+# commit/tag/branch
+ARG QEMU_VIRTIOFS_TAG
+ARG PREFIX
+
+WORKDIR /root/qemu-virtiofs
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y \
+	    autoconf \
+	    automake \
+	    bc \
+	    bison \
+	    cpio \
+	    flex \
+	    gawk \
+	    libaudit-dev \
+	    libcap-dev \
+	    libcap-ng-dev \
+	    libdw-dev \
+	    libelf-dev \
+	    libglib2.0-0 \
+	    libglib2.0-dev \
+	    libglib2.0-dev git \
+	    libltdl-dev \
+	    libpixman-1-dev \
+	    libseccomp-dev \
+	    libtool \
+	    pkg-config \
+	    pkg-config \
+	    python \
+	    python-dev \
+	    rsync \
+	    zlib1g-dev
+
+RUN cd  .. && git clone "${QEMU_VIRTIOFS_REPO}" qemu-virtiofs
+RUN git checkout "${QEMU_VIRTIOFS_TAG}"
+ADD scripts/configure-hypervisor.sh /root/configure-hypervisor.sh
+# Currently the tag does not provide a VERSION file for qemu and as this is
+# necessary to run the configure-hypervisor.sh, we are passing the current
+# qemu version that is supported
+RUN echo 4.1.0 > VERSION
```

Comment:
@egernst and @jodh-intel , I did this temporary workaround, do you guys have a better idea of how to deal with this? Once again thanks for all the feedback ...

Reply:
weird, there is the VERSION file https://gitlab.com/virtio-fs/qemu/blob/virtio-fs-v0.3/VERSION